crates/ibc/src/component/client.rs

```diff
@@ -265,14 +265,14 @@ pub trait StateReadExt: StateRead {
     async fn get_client_type(&self, client_id: &ClientId) -> Result<ClientType> {
         self.get_proto(&state_key::client_type(client_id))
             .await?
-            .ok_or_else(|| anyhow::anyhow!("client not found"))
+            .context("could not find client type")
```

Comment:
What about "could not find client type for {client_id}"?

Reply:
good idea, done